spec/06_datadog_propagation_spec.lua

```diff
@@ -0,0 +1,237 @@
+local extract_datadog = require "kong.plugins.ddtrace.datadog_propagation".extract
+local inject_datadog = require "kong.plugins.ddtrace.datadog_propagation".inject
+local new_span = require "kong.plugins.ddtrace.span".new
+
+local function get_header_builder(headers)
```

Comment:
Does it get a header-builder, or is it a builder that produces a get-header? It's the latter. Consider `make_getter` instead.